src/dioptra/restapi/v1/workflows/errors.py

```diff
@@ -39,3 +43,10 @@ def handle_experiment_does_not_exist_error(error):
             "message": "Not Found - The experiment associated with the job does not "
             "exist"
         }, 404
+
+    @api.errorhandler(EntrypointWorkflowValidationIssue)
+    def handle_entrypoint_workflow_validation_error(error):
+        issues = error.args
+        print (issues)
+        message = "\n".join(str(issue) for issue in issues)
```

Comment:
We should create a "detail" dictionary that extracts the relevant information from task engine issues.

See Harold's error handling refactor: https://github.com/usnistgov/dioptra/blob/a66cad4d524c733a936ec0afe6f6242d4a96c137/src/dioptra/restapi/errors.py#L341

This lets use return more structured information instead of just string-ifiying the issues directly.